src/main/java/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package command;
+import duke.*;
+import task.*;
+
+/**
+ * Command represents a command.
+ *
+ * @author Ho Wen Zhong
+ */
+public abstract class Command {
+
+    /**
+     * Executes the Command.
+     *
+     * @param tasks Current TaskList.
+     * @param ui Ui object.
+     * @param storage Storage object.
+     */
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage);
+
+    /**
+     * Returns whether the Command exits the program.
+     *
+     * @return Boolean indicating if Command exits program.
+     */
+    public abstract boolean isExit();
```

Comment:
Consider making this a concrete method instead of abstract since only 1 command will return true. So just override it for ExitCommand and you don't have to implement it in all the other commands.